cedar/.htaccess

```diff
@@ -0,0 +1,3 @@
+RewriteEngine on
+
+RewriteRule ^cedar(.*)$ http://cedar.openmadrigal.org/showExperiment/$1 [R=302,L]
```

Comment:
This is duplicating the current directory "cedar" and would match /cedar/cedar*.  You probably mean to do "^(.*)$".